/******************************************************************************
 *
 *  Copyright (c) 2014-2015, 2020-2021 The Linux Foundation. All rights reserved
 *  Copyright (C) 2003-2012 Broadcom Corporation
 *  Copyright (C) 2008, 2011-2012, 2016-2018 The Android Open Source Project
 *  Copyright (C) 2014 Google, Inc
 *
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  you may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at:
 *
 *  http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions and
 *  limitations under the License.
 *
 ******************************************************************************/

From 50d4837f01b5d3027280d283c0cbb3b1c26a6a1e Mon Sep 17 00:00:00 2001
From: Subramanian Srinivasan <quic_subrsrin@quicinc.com>
Date: Fri, 13 May 2022 15:10:24 -0700
Subject: [PATCH] Update subrate min values for Enhanced connection update
 modes

Update subrate min values for Balanced and Low power
modes for Enhanced connection update procedure.

Change-Id: I58b469def26f6327f55be8983a0185fccae03ae1
---
 res/values/config.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values/config.xml b/res/values/config.xml
index 689361c39..322c898c8 100644
--- a/res/values/config.xml
+++ b/res/values/config.xml
@@ -108,9 +108,9 @@
          balanced and low power subrate request configurations. -->
     <integer name="subrate_mode_high_priority_min_subrate">2</integer>
     <integer name="subrate_mode_high_priority_max_subrate">4</integer>
-    <integer name="subrate_mode_balanced_min_subrate">5</integer>
+    <integer name="subrate_mode_balanced_min_subrate">4</integer>
     <integer name="subrate_mode_balanced_max_subrate">7</integer>
-    <integer name="subrate_mode_low_power_min_subrate">8</integer>
+    <integer name="subrate_mode_low_power_min_subrate">4</integer>
     <integer name="subrate_mode_low_power_max_subrate">10</integer>
 
     <!-- Specifies max latency parameters for high priority, balanced
-- 
2.25.1

